addon/server.js

```diff
@@ -16,6 +16,11 @@ export default class EmberServer extends Server {
     }
 
     super(options);
+
+
+    if (window.QUnit && window.QUnit.config.mirageLogging) {
```

Comment:
No need to require `window.QUnit` here, something like this would do:

```suggestion
    if (typeof QUnit !== 'undefined' && QUnit.config.mirageLogging) {
```

Reply:
Also, FWIW, if you searched for a query param instead of QUnit.config.mirageLogging this would also work for ember-mocha users.

Something like:

```suggestion
    if (typeof location !== 'undefined' && location.search.indexOf('mirageLogging') > -1) {
```

Reply:
good idea, thanks 👍